haxepunk/math/MathUtil.hx

```diff
@@ -105,13 +104,11 @@ class MathUtil
 	 * @param	anchor		The anchor object.
 	 * @param	distance	The max distance object can be anchored to the anchor.
 	 */
-	public static inline function anchorTo(object:Position, anchor:Position, distance:Float = 0)
+	public static inline function anchorTo(object:Vector2, anchor:Vector2, distance:Float = 0)
 	{
-		point.x = object.x - anchor.x;
-		point.y = object.y - anchor.y;
+		point = object - anchor;
```

Comment:
It looks like this is allocating a new object each time. The in place modification is preferable even if it's uglier.

Reply:
Yeah, I'll revert that one.